cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/http/handlers/SourceControlManagementHttpHandlerTests.java

```diff
@@ -62,9 +74,32 @@ public static void beforeClass() throws Throwable {
 
   @Before
   public void before() {
+    Mockito.reset(getInjector().getInstance(SourceControlManagementService.class));
     setSCMFeatureFlag(true);
   }
 
+  protected static void initializeAndStartServices(CConfiguration cConf) throws Exception {
+    initializeAndStartServices(cConf, new AbstractModule() {
+      @Override
+      protected void configure() {
+        bind(UGIProvider.class).to(CurrentUGIProvider.class);
+        bind(MetadataSubscriberService.class).in(Scopes.SINGLETON);
+      }
+
+      @Provides
+      @Singleton
+      public SourceControlManagementService createLifeCycleService(CConfiguration cConf,
```

Comment:
Why is this called `createLifeCycleService` and not `SourceControlManagementService`?